store/pebble/pebble.go

```diff
@@ -80,8 +80,11 @@ func (s *store) Get(mh multihash.Multihash) ([]indexer.Value, bool, error) {
 		log.Errorw("can't find multihash", "err", err)
 		return nil, false, err
 	}
-	vks, err := s.vcodec.UnmarshalValueKeys(vkb)
+	vkbcpy := make([]byte, len(vkb))
+	copy(vkbcpy, vkb)
 	_ = vkbClose.Close()
+
+	vks, err := s.vcodec.UnmarshalValueKeys(vkbcpy)
```

Comment:
I would prefer that `s.vcodec.UnmarshalValueKeys` is immune to passing it a temporary buffer. Having the unmarshalling code depend on the perpetual validity of the buffer passed in is not desirable as that memory could in other use cases be part of a much larger body of memory that then would not get freed.